tests/Unit/DataStructures/DataBox/TestHelpers.hpp

```diff
@@ -0,0 +1,38 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+#pragma once
+
+#include "tests/Unit/TestingFramework.hpp"
+
+#include <string>
+
+#include "DataStructures/DataBox/DataBoxTag.hpp"
+#include "Utilities/TypeTraits.hpp"
+
+namespace test {
+
+namespace db {
+
+namespace detail {
+CREATE_IS_CALLABLE(name)
+}
+
+template <typename Tag>
+void test_simple_tag(const std::string& expected_name) {
+  static_assert(cpp17::is_base_of_v<::db::SimpleTag, Tag> and
+                    not cpp17::is_base_of_v<::db::ComputeTag, Tag>,
+                "A simple tag must be derived from "
+                "db::SimpleTag, but not db::ComputeTag");
+  static_assert(not cpp17::is_same_v<Tag, typename Tag::type>,
+                "A type cannot be its own tag.");
+  CHECK(::db::tag_name<Tag>() == expected_name);
+  if (detail::is_name_callable_v<Tag>) {
+    INFO("Do not define name for Tag '" << ::db::tag_name<Tag>() << "',");
```

Comment:
I'm worried that with using `INFO` here you will collect all the messages for all tags that have a `name()` function. Have you verified that the scoping applies correctly?

Reply:
Yes, I got one INFO per error message

Reply:
Cool, cool :P